### PR TITLE
Created users controller and fixed navbar login / home root issue.

### PR DESCRIPTION
**Pull request by DenzelBraithwaite**
Skipped requiring authentication on the home page and the show page for pokemons.
Please review @MarlAndre , try signing out and signing up / logging in.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -6,11 +6,20 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+puts "Releasing Pokemons to the wild"
 Pokemon.destroy_all
+temp_pokmeon_user_id = 1
 
-Pokemon.create!(
```
Comment (MarlAndre) [inline db/seeds.rb:11]: Won't work, need to change it

Comment (MarlAndre) [reply db/seeds.rb:11]: Seed file won't work, but won't cause merging issues

**Review by MarlAndre — APPROVED** [review]
@DenzelBraithwaite make a branch to fix seeding file

File: db/seeds.rb (hunk 1 above)
Comment (DenzelBraithwaite) [reply db/seeds.rb:11]: True, I will make a branch to fix this, thanks

**Pull request merged**